documentation/modules/ROOT/pages/integrations/cloudevents.adoc

```diff
@@ -97,6 +101,9 @@ Alternatively, Avro can be used as content type for the `data` attribute:
   "iodebeziumtxId" : "578",
   "iodebeziumlsn" : "33227720",
   "iodebeziumxmin" : null,
+  "iodebeziumtxid": "589",
```

Comment:
There is `iodebeziumtxId` already (see above). That's really what I'm a bit concerned about -- the redundancy of this new info. I don't have a good answer right now. Maybe we should deprecate and eventually remove the current txId attribute in the source block?

Reply:
@gunnarmorling I don't think that it is necessary to remove the redundancy. Don't forget that transaction metadata are optional. So I'd keep it in source info.
In fact this looks like a problem only in flattened format and in this case I propose to just document it and/or change `iodebeziumtx` to a different one so the difference is more visible, maybe `iodebeziumTxn`?